src/com/redomar/game/entities/Player.java

```diff
@@ -38,6 +43,8 @@ public Player(LevelHandler level, int x, int y, InputHandler input,
 		this.shirtCol = shirtCol;
 		this.colour = Colours.get(-1, 111, shirtCol, faceCol);
 		fireRate = Small.FIRE_RATE;
+		// Perfectly matches the text over the head
+		nameOffset = (userName.length() / 2) * fontCharSize - ((userName.length() & 1) == 0 ? fontCharSize / 2 : 0);
```

Comment:
Might consider splitting the equation from the ternary for readability.

Reply:
You are right about that, didn't observe its over 80 characters long.